internal/service/ipam/api_server_resource_test.go

```diff
@@ -0,0 +1,1089 @@
+				Config: testAccServerDhcpConfig(name, true, true, true, 50, 60),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.allow_unknown", "true"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.allow_unknown_v6", "true"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.ignore_client_uid", "true"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.lease_time", "50"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.lease_time_v6", "60"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccServerDhcpConfig(name, false, false, false, 55, 65),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.allow_unknown", "false"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.allow_unknown_v6", "false"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.ignore_client_uid", "false"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.lease_time", "55"),
+					resource.TestCheckResourceAttr(resourceName, "dhcp_config.lease_time_v6", "65"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccServerResource_GssTsigFallback(t *testing.T) {
+	var resourceName = "bloxone_dhcp_server.test_gss_tsig_fallback"
+	var v ipam.IpamsvcServer
+	var name = acctest.RandomNameWithPrefix("dhcp-server")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccServerGssTsigFallback(name, "true"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "gss_tsig_fallback", "true"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccServerGssTsigFallback(name, "false"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckServerExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "gss_tsig_fallback", "false"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccServerResource_HeaderOptionFilename(t *testing.T) {
+	var resourceName = "bloxone_dhcp_server.test_header_option_filename"
+	var v ipam.IpamsvcServer
+	var name = acctest.RandomNameWithPrefix("dhcp-server")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccServerHeaderOptionFilename(name, "TEST_HEADER_OPTION_FILENAME"),
```

Comment:
Would be nice if you could replace this `TEST_HEADER_OPTION_FILENAME` with a filename

Reply:
done